src/libsync/theme.cpp

```diff
@@ -188,6 +198,26 @@ QIcon Theme::themeIcon(const QString &name, bool sysTray) const
     return cached;
 }
 
+QString Theme::themeImagePath(const QString &name, int size, bool sysTray) const
+{
+    const auto flavor = (!isBranded() && sysTray) ? systrayIconFlavor(_mono) : QLatin1String("colored");
+
+    // branded cient may have several sizes of the same icon
+    const QString filePath = (isBranded() && size > 0)
+            ? QString::fromLatin1(":/client/theme/%1/%2-%3").arg(flavor).arg(name).arg(size)
+            : QString::fromLatin1(":/client/theme/%1/%2").arg(flavor).arg(name);
+
+    const QString brandedExtension = ".png";
```

Comment:
I think I'd have instead the full png name here. This way it avoids the concatenation twice later on and it'd feel slightly more readable to me.